src/main/kotlin/dev/arbjerg/lavalink/client/loadbalancing/regionFiltering.kt

```diff
@@ -55,40 +72,46 @@ object RegionGroup {
         "ASIA" -> ASIA
         "EUROPE" -> EUROPE
         "US" -> US
+        "SOUTH_AMERICA" -> SOUTH_AMERICA
+        "AFRICA" -> AFRICA
         else -> throw IllegalArgumentException("No region constant: $region")
     }
 }
 
 // TODO In case no exact server match, should it look for the closest node in that same region?
-enum class VoiceRegion(val id: String, val visibleName: String) {
-    BRAZIL("brazil", "Brazil"),
-    HONGKONG("hongkong", "Hong Kong"),
-    INDIA("india", "India"),
-    JAPAN("japan", "Japan"),
-    ROTTERDAM("rotterdam", "Rotterdam"),
-    RUSSIA("russia", "Russia"),
-    SINGAPORE("singapore", "Singapore"),
-    SOUTH_AFRICA("southafrica", "South Africa"),
-    SIDNEY("sidney", "Sidney"),
-    US_CENTRAL("us-central", "US Central"),
-    US_EAST("us-east", "US East"),
-    US_SOUTH("us-south", "US South"),
-    US_WEST("us-west", "US West"),
-
-    UNKNOWN("", "Unknown");
+enum class VoiceRegion(val id: String, val subregions: HashSet<String>, val visibleName: String) {
+    BRAZIL("brazil", hashSetOf("buenos-aires", "santiago"), "Brazil"),
+    HONGKONG("hongkong", hashSetOf(), "Hong Kong"),
+    INDIA("india", hashSetOf(), "India"),
+    JAPAN("japan", hashSetOf("south-korea"),"Japan"),
+    ROTTERDAM("rotterdam", hashSetOf("frankfurt", "stockholm", "bucharest", "milan", "madrid"),"Rotterdam"),
+    RUSSIA("russia", hashSetOf(),"Russia"),
+    SINGAPORE("singapore", hashSetOf(),"Singapore"),
+    SOUTH_AFRICA("southafrica", hashSetOf("dubai", "tel-aviv"),"South Africa"),
+    SYDNEY("sydney", hashSetOf(), "Sydney"),
+    US_CENTRAL("us-central", hashSetOf(), "US Central"),
+    US_EAST("us-east", hashSetOf("newark"), "US East"),
+    US_SOUTH("us-south", hashSetOf("atlanta"), "US South"),
+    US_WEST("us-west", hashSetOf("seattle", "santa-clara"), "US West"),
+
+    UNKNOWN("", hashSetOf(), "Unknown");
 
     companion object {
         @JvmStatic
         fun fromEndpoint(endpoint: String): VoiceRegion {
-            val endpointRegex = "^([a-z\\-]+)[0-9]+.*:443\$".toRegex()
```

Comment:
Mah regex :(
https://going-g.host/Vfc8Q1TCty.png

(/j)

Reply:
I copied over from some other stuff I implemented, but tested the regex out with a handful of examples and it works fine as well :)